docs/Examples/README.md

```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+* [Basic usage](Basic.md)
```

Comment:
We already have ReactiveCocoa and RxSwift examples in docs.
Moved them now to Examples folder as well. And added to Readme, just below "basic usage".